GenderPayGap.WebUI/Views/AccountCreation/UserNotFoundErrorPage.cshtml

```diff
@@ -0,0 +1,38 @@
+﻿@model GenderPayGap.WebUI.Models.AccountCreation.CreateUserAccountViewModel
+@{
+    ViewBag.Title = "User not found - Gender pay gap reporting service";
+    Layout = "~/Views/GovUkFrontend/GovUkFrontendLayout.cshtml";
+}
+
+@section BeforeMain {
+    @{ await Html.RenderPartialAsync("/Views/ReportingStepByStep/ReportingPageHeader.cshtml"); }
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <h1 class="govuk-heading-xl">User not found</h1>
+    </div>
+</div>
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <p class="govuk-body">
+            No user was found for that verification code.
+        </p>
```

Comment:
I wonder if this could be any clearer.

I'm thinking:

> **We could not confirm your email address**
> We could not find a user with that confirmation code.
> If you pasted the web address, check you copied the entire address.
> Or please _create a new account_

Let's talk to a few people to see what they think (although it's a edge case so it's not so important)

Reply:
I agree we should talk to people about this page. But I've made your suggested changes for now so that we can keep moving forward.